internal/grpc/services/appprovider/appprovider.go

```diff
@@ -154,50 +146,41 @@ func (s *service) OpenFileInAppProvider(ctx context.Context, req *providerpb.Ope
 		return nil, err
 	}
 
-	httpReq, err := rhttp.NewRequest(ctx, "GET", wopiurl+"wopi/iop/open", nil)
+	httpReq, err := rhttp.NewRequest(ctx, "GET", s.conf.WopiURL+"wopi/iop/open", nil)
```

Comment:
Can we take care of missing trailing slashes here? Something like [this](https://github.com/cs3org/reva/blob/master/pkg/ocm/invite/manager/json/json.go#L210)

Reply:
Sure, makes sense

Reply:
How does it look like? I note that the block with the HTTP request to `/wopi/cbox/endpoints` should be moved elsewhere, in some kind of init of the service. Maybe I could move it to the `init()` function? or where else?

Reply:
Answering to myself, in fact I found a way to do that with a separate function. This appprovider will eventually be the WOPI driver, so it's OK to have it like this - commit to follow.